packages/styled-base/types/index.d.ts

```diff
@@ -24,127 +23,88 @@ export {
   ObjectInterpolation
 } from '@emotion/serialize'
 
-export { ComponentSelector, Interpolation }
-
-type JSXInEl = JSX.IntrinsicElements
-
-export type WithTheme<P, T> = P extends { theme: infer Theme }
-  ? P & { theme: Exclude<Theme, undefined> }
-  : P & { theme: T }
+export { ComponentSelector, Interpolation, PropsOf, Omit }
```

Comment:
does `Omit` have to exported from here?

Reply:
I thought it would be preferable than duplicating it like it was before. It is used by emotion-theming